src/core/server/elasticsearch/client/configure_client.ts

```diff
@@ -35,9 +35,15 @@ export const configureClient = (
 };
 
 const addLogging = (client: Client, logger: Logger, logQueries: boolean) => {
-  client.on('response', (err, event) => {
-    if (err) {
-      logger.error(`${err.name}: ${err.message}`);
+  client.on('response', (error, event) => {
+    if (error) {
+      const errorMessage =
+        // error details for response errors provided by elasticsearch
+        error.name === 'ResponseError'
+          ? `[${event.body.error.type}]: ${event.body.error.reason}`
```

Comment:
TODO: in a separate task we should extend this information with response method & path